Makefile

```diff
@@ -33,6 +33,22 @@ update-beats:
 	@$(MAKE) update
 	@echo --- Use this commit message: Update beats framework to `cat vendor/vendor.json | python -c 'import sys, json; print([p["revision"] for p in json.load(sys.stdin)["package"] if p["path"] == "github.com/elastic/beats/libbeat/beat"][0][:7])'`
 
+.PHONY: check-headers
```

Comment:
did you try to come up with a way to avoid duplicating these?  I haven't been able to figure one out.  These warnings are a bit annoyting:

```
Makefile:39: warning: overriding commands for target `check-headers'
_beats/libbeat/scripts/Makefile:128: warning: ignoring old commands for target `check-headers'
Makefile:47: warning: overriding commands for target `add-headers'
_beats/libbeat/scripts/Makefile:135: warning: ignoring old commands for target `add-headers'```

Reply:
no :( 
agreed is a bit annoying, we would have define our own make apm-fmt or something like that with new renamed targets.
or, open a pr in the licenser to not complain if the directory arg doesn't exist and then update `libbeat/scripts/Makefile`, but i am guessing that the licenser complains with missing directories for a good reason.

Reply:
Fair enough, I agree it shouldn't hold this up.